kubernetes/windows/main.ps1

```diff
@@ -64,19 +64,11 @@ function Set-EnvironmentVariables {
         $wsID = Get-Content /etc/omsagent-secret/WSID
     }
 
-    # Set DOMAIN
+    # Set WSID
     [System.Environment]::SetEnvironmentVariable("WSID", $wsID, "Process")
     [System.Environment]::SetEnvironmentVariable("WSID", $wsID, "Machine")
 
-    $wsKey = ""
```

Comment:
Grace, There is another place this is going thru ENV (one agent). Please keep this work item active (or close & recreate a new one), just so that we can track it.

Reply:
Thanks Vishwa, will do